plugins/plugin-java/che-plugin-java-ext-lang-server/src/test/java/org/eclipse/che/plugin/java/server/jdt/quickfix/AssistQuickFixTest.java

```diff
@@ -696,12 +696,12 @@ public void testAssignToLocal10() throws Exception {
     buf.append("package test1;\n");
     buf.append("public class E {\n");
     buf.append("    public void foo() {\n");
-    buf.append("        System.getProperties()\n");
```

Comment:
@sleshchenko looks like this class is changed by mistake

Reply:
Good catch!